docs/release-process.md

```
@@ -204,6 +204,18 @@ If all was successful:
 
 * Add all files including signingkey.asc and the gpg pub keys to GitHub release page
 
+* After the GitHub assets are uploaded and the webpage pubkeys/downloads are deployed, run the release readiness check:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)